src/components/Work/Tabs/Cite.js

```diff
@@ -31,27 +31,27 @@ const flexTitle = css`
 `;
 
 const TabsCite = ({ item }) => {
-  const {
-    administrativeMetadata: { libraryUnit }, // = "Library Division"
-    collection,
-    createDate: [date] = "",
-    descriptiveMetadata,
-    id,
-    termsOfUse,
-  } = item;
+  const { collection, createDate: [date] = "", descriptiveMetadata, id } = item;
 
-  const { ark, identifier, license, title } = descriptiveMetadata;
+  const { ark, identifier, license, title, termsOfUse } = descriptiveMetadata;
 
   const [copied, setCopied] = useState();
 
   const nul = "Northwestern University Libraries";
   const itemLink = `${window.location.origin}/items/${id}`;
   const today = new Date().toDateString();
-  const collectionTitle = collection ? collection.title : "";
+  const collectionTitle =
+    Object.keys(collection).length === 0 ? "" : collection.title;
+  const libraryUnitLabel = item.administrativeMetadata.libraryUnit
+    ? item.administrativeMetadata.libraryUnitLabel
+    : "";
```

Comment:
Since `administrativeMetadata` will always be present on `item`, you may be able to shorten this with a `item.administrativeMetadata.libraryUnit || ""`?

Reply:
I think that's needed because libraryUnit can be null. But there is a mistake there, the first case should be `item.administrativeMetadata.libraryUnit.label` not `item.administrativeMetadata.libraryUnitLabel`

```
"administrativeMetadata": {
      "libraryUnit": null,
...
```